subsys/bluetooth/host/gatt.c

```diff
@@ -2542,20 +2594,140 @@ int bt_gatt_notify_cb(struct bt_conn *conn,
 int bt_gatt_notify_multiple(struct bt_conn *conn, uint16_t num_params,
 			    struct bt_gatt_notify_params *params)
 {
-	int i, ret;
+	int i;
+	int err;
+	size_t total_len = 0;
+	struct net_buf *buf;
+	struct notify_data data;
```

Comment:
Scope of `struct notify_data data;` can be reduced by moving it to the for-loop in line 2702